packages/caliper-tests-integration/generator_tests/myWorkspace/src/marbles/node/package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "marbles",
+	"version": "1.0.0",
+	"description": "marbles chaincode implemented in node.js",
+	"engines": {
+		"node": ">=8.4.0",
+		"npm": ">=5.3.0"
+	},
+	"scripts": {
+		"start": "node marbles.js"
```

Comment:
general comment - is it necessary to include the smart contracts in the workspace? Is it possible to link direct to existing s-contracts outside the workspace, or is that a current limitation of caliper (if so we might want to fix that at some point ... but not for this)